setup.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='quickcache',
+    version='0.0.1',
```

Comment:
haha, well, to be fair the ripped-out version is not that battle tested. I think I'm more on the semantic side of the [semantic vs. romantic](http://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/) versioning debate, but since no one's using this yet, it doesn't much matter to me. Maybe I could bump it to 1.0.0 once it's been broken out for a while.